### PR TITLE
maint(fix serial task for release checks)

### PR DESCRIPTION
**Pull request by sheenaajay**

### CODE REVIEW

File: lib/puppet_litmus/rake_tasks.rb [hunk 1]
```diff
@@ -377,6 +377,9 @@ def run_local_command(command)
         # The host_name might be localhost or an IP or a DNS-resolvable node.
         prefix = 'litmus:acceptance:'
         tasks = Rake::Task.tasks.select { |task| task.name =~ %r{^#{prefix}.+:\d+$} }
+        if tasks.empty?
+          tasks = Rake::Task.tasks.select { |task| task.name =~ %r{delivery.puppetlabs.net$$}; }
```
Comment (michaeltlombardi) [inline lib/puppet_litmus/rake_tasks.rb]: Ahhhhhhhhhhh I understand now why we were finding them locally and failing via vmpooler! Whoops.

**Comment by DavidS** [comment]
what is the format for those rake task names? how will that specific pattern interact with other backends?

File: lib/puppet_litmus/rake_tasks.rb (hunk 1 above)
Comment (florindragos) [inline lib/puppet_litmus/rake_tasks.rb]: Wouldn't it make more sense to extend the regex so it matches also for the vmpooler hostnames?

Comment (tphoney) [reply lib/puppet_litmus/rake_tasks.rb]: why is it not everything but localhost ?

Comment (florindragos) [reply lib/puppet_litmus/rake_tasks.rb]: something like `^#{prefix}.\S+` ?

Comment (tphoney) [reply lib/puppet_litmus/rake_tasks.rb]: no tests :(

Comment (sheenaajay) [reply lib/puppet_litmus/rake_tasks.rb]: @tphoney oops adding the tests now

Comment (sheenaajay) [reply lib/puppet_litmus/rake_tasks.rb]: @florindragos sure let me try that. Thank you.

Comment (florindragos) [reply lib/puppet_litmus/rake_tasks.rb]: actually no, this would include `serial` and `parallel`

**Comment by sheenaajay** [comment]
@DavidS sorry for the late reply.
These are the formats of rake tasks
Run against vmpooler
rake litmus:acceptance:oag4qztd04qnagy.delivery.puppetlabs.net
Run against docker
rake litmus:acceptance:localhost:2222
@tphoney fixed the tests to run parallelly so we dont have to run them serially now.
Thanks a lot for looking into the PR @michaeltlombardi @florindragos @tphoney @DavidS

**Pull request closed without merging**